src/main/java/com/github/black0nion/blackonionbot/systems/JoinLeaveSystem.java

```diff
@@ -64,7 +65,13 @@ public void onGuildMemberJoin(final @NotNull GuildMemberJoinEvent event) {
 			final TextChannel channel = guild.getTextChannelById(id);
 			if (channel == null) return;
 			final File file = generateImage(Color.BLACK, author, guild, DrawType.JOIN);
-			channel.sendMessage(guild.getJoinMessage().replace("%userid%", author.getAsMention()).replace("%guildid%", guild.getEscapedName())).addFiles(FileUpload.fromData(file, "welcome.png")).queue();
```

Comment:
Please re-add this line, it's supposed to be a welcome message _but on the server_

Reply:
what do you mean

Reply:
the deleted line sends a message in a channel in the guild
the code you added is to send a DM to the user to make them aware of rules, links and that stuff
my plan was for both to co-exist